tools/nuget/nuget.proj

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation
+     SPDX-License-Identifier: MIT
+-->
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <OutputPath>$(MSBuildThisFileDirectory)..\..\$(Platform)\$(Configuration)</OutputPath>
+    <PackageId>eBPF-for-Windows.Extensions</PackageId>
+    <IsPackable>true</IsPackable>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>eBPF for Windows Contributors</Authors>
+    <Owners>eBPF for Windows Contributors</Owners>
+    <Title>eBPF for Windows Extensions</Title>
+    <PackageDescription>eBPF for Windows Extensions</PackageDescription>
+    <PackageVersion>$(eBPFExtensionsVersionMajor).$(eBPFExtensionsVersionMinor).$(eBPFExtensionsVersionRevision)</PackageVersion>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/Microsoft/ntosebpfext</PackageProjectUrl>
+    <PackageTags>native ebpf</PackageTags>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="eBPF-for-Windows" />
+  </ItemGroup>
+
+  <Target Name="_Get_NetNativeContent" BeforeTargets="PrepareForBuild;_IntermediatePack">
+    <ItemGroup>
+      <None Include="README.md" PackagePath="." Pack="true" />
+      <None Include="..\..\$(Platform)\$(Configuration)\ntosebpfext.sys" PackagePath="build\native\bin" Pack="true" />
```

Comment:
I probably need to include more files here - the INF files?  The export program info exes?  I'm not sure of the full set yet.

Reply:
The INF files aren't needed, but the nuget should probably be include the export program info binaries.